build2slack.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# Copyright (c) 2018 Arista Networks, Inc.  All rights reserved.
+# Arista Networks, Inc. Confidential and Proprietary.
+
+import argparse
+import base64
+import json
+import os
+import requests
+import subprocess
+import yaml
+
+
+GERRIT_CHANGE_COMMIT_MESSAGE_ENV = "GERRIT_CHANGE_COMMIT_MESSAGE"
+GERRIT_CHANGE_NUMBER_ENV = "GERRIT_CHANGE_NUMBER"
+GERRIT_CHANGE_URL_ENV = "GERRIT_CHANGE_URL"
+GERRIT_PATCHSET_NUMBER_ENV = "GERRIT_PATCHSET_NUMBER"
+GERRIT_REVISION_ENV = "GERRIT_PATCHSET_REVISION"
+PACKAGE_SETTINGS_FILE = 'package.yml'
+SLACK_WEBHOOK_ENV = "SLACK_WEBHOOK"
+
+
+def getEnv(name):
+  value = os.environ.get(name)
+  assert value, name + " is not set"
+  return value
+
+
+def gerritRevision():
+  rev = getEnv(GERRIT_REVISION_ENV)
+  diff = subprocess.check_output(["git", "diff-tree", "--no-commit-id", "--name-only", "-r", rev])
+  return diff.split("\n")
+
+
+def dirs(files):
+  return set([os.path.dirname(f) for f in files])
```

Comment:
Inner `[` and `]` are not necessary and should be removed.

Reply:
👍